backend/spec/controllers/spree/admin/store_credits_controller_spec.rb

```diff
@@ -93,83 +153,135 @@
           store_credit: {
             amount: updated_amount,
             category_id: a_credit_category.id
+          category_id: b_credit_category.id
```

Comment:
There are also some errors that are due to the `StoreCreditsController` being a `ResourceController` in Solidus, looking into fixing those as well.